raster/raster-function/src/main/java/com/esri/samples/raster_function/RasterFunctionSample.java

```diff
@@ -56,10 +57,14 @@ public void start(Stage stage) {
       stage.setScene(scene);
       stage.show();
 
-      // create a map with dark canvas vector basemap
-      ArcGISMap map = new ArcGISMap(Basemap.createDarkGrayCanvasVector());
+      // authentication with an API key or named user is required to access basemaps and other location services
+      String yourAPIKey = System.getProperty("apiKey");
+      ArcGISRuntimeEnvironment.setApiKey(yourAPIKey);
 
-      // add the map to a map view
+      // create a map with a dark gray basemap style
```

Comment:
I won't comment on any other instances, before I make a fool of myself!
```suggestion
      // create a map with the dark gray basemap style
```